lib/vagrant-registration.rb

```diff
@@ -15,6 +15,8 @@ def self.source_root
       @source_root ||= Pathname.new(File.expand_path("../../", __FILE__))
     end
 
+    # Temporally load the extra capability files for Fedora
+    load(File.join(self.source_root, 'plugins/guests/fedora/plugin.rb'))
```

Comment:
We should still consider loading these files according to the current machine's OS (guest).